src/pages/workspace/WorkspaceInviteMessagePage.tsx

```diff
@@ -83,7 +84,7 @@ function WorkspaceInviteMessagePage({
         workspaceInviteMessageDraft ||
         // policy?.description can be an empty string
         // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-        policy?.description ||
+        Parser.htmlToMarkdown(policy?.description ?? '') ||
```

Comment:
We got an issue where an HTML is shown in the invite message page. This happens when you set the workspace description with a markdown. The description is saved as HTML, so we need to convert it back to markdown.

Reply:
Before our PR, we covert the HTML to markdown in
https://github.com/Expensify/App/blob/3d465c345e9497bce7448233c66a1b3083c82ee4/src/pages/workspace/WorkspaceInviteMessagePage.tsx#L92-L94

but that has its flaw too because it will convert the message draft HTML to markdown. For example, if user type `<b>test</b>`, go back and reopen the page, they will see `*b*`.

So, what we did already correct, but we missed moving the htmlToMarkdown to the workspace description.